exchanges/binance/type_convert.go

```diff
@@ -344,3 +346,12 @@ func (a *wsListStatus) UnmarshalJSON(data []byte) error {
 	a.Data.TransactionTime = aux.Data.TransactionTime.Time()
 	return nil
 }
+
+// formatSymbol formats the given pair to a string suitable for exchange API requests
+func (b *Binance) formatSymbol(pair currency.Pair) (string, error) {
```

Comment:
This is just a comment: We have an open PR: #530 which will introduce USDT Margined Futures and Coin Margined Futures; coin margined futures requires a different request format compared to spot and USDT margined futures. This might need to be upgraded in the future when this gets merged and asset.Item type passed in with the currency pair from wrapper to these functions that require formatting. This is not exactly ideal for what you are doing now.

Reply:
For now since we only support spot and margin assets which use the same request format, this is fine as is. The support for different asset request formatting can implemented in #530

Reply:
With each (non-interface) API call you implicitly have the context whether the call is for spot or margin or other asset types - thus I'm not worried about losing information between the wrapper/API boundary.
That's also why I kept it non-exported - it should only be used in the context that is relevant - currently for Spot (and Margin) only.
I added a comment for now.
If you wish, I'd be happy to generalise it to:
`func (b *Binance) formatSymbol(pair currency.Pair, a asset.Type) (string, error)`
to make it useful for future calls?

Reply:
I have no issue with keeping it as it is. We can do that later. Thanks for the continued clarification.